nilearn/maskers/base_masker.py

```diff
@@ -55,21 +54,16 @@ def _filter_and_extract(
     """
     if memory is None:
         memory = Memory(location=None)
-    # Since the calling class can be any *Nifti*Masker, we look for exact type
-    if verbose > 0:
-        class_name = enclosing_scope_name(stack_level=10)
```

Comment:
Only place in the codebase where `enclosing_scope_name` is used: consider removing?

Or integrate into the logger module?

TODO: check that the output printed by the masker loggers still make sense after this change.